.github/workflows/neon_extra_builds.yml

```diff
@@ -122,7 +125,31 @@ jobs:
         run: cargo build --all --release --timings
 
       - name: Upload the build stats
-        uses: actions/upload-artifact@v3
+        id: upload-stats
+        env:
+          BUCKET: neon-github-public-dev
+          SHA: ${{ github.event.pull_request.head.sha || github.sha }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_DEV }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_KEY_DEV }}
+        run: |
+          REPORT_URL=https://${BUCKET}.s3.amazonaws.com/build-stats/${SHA}/${GITHUB_RUN_ID}/cargo-timing.html
+          aws s3 cp --recursive --only-show-errors ./target/cargo-timings/ "s3://${BUCKET}/build-stats/${SHA}/${GITHUB_RUN_ID}/"
+          echo "report-url=${REPORT_URL}" >> $GITHUB_OUTPUT
```

Comment:
can you set a TTL for files to be one week?

Reply:
https://aws.amazon.com/blogs/aws/amazon-s3-object-expiration/ says that in the management console, I can set up those per prefix.

So, I guess I can. Will do that after the merge.

Reply:
Set the retention for 60 days, just in case somebody wants to draw charts based on that data.
Check out the [corresponding bucket settings](https://s3.console.aws.amazon.com/s3/management/neon-github-public-dev/lifecycle/view?region=eu-central-1&id=Rust+build+stats+cleanup) if you want to verify me.

Both report files seem identical, so I've changed the code to copy only the `cargo-timing.html` one.
That leaves us with 408.2 KiB per build run (`main` + manual invocations via tags).
Feels ok to leave these for two months.

```
work/neon/neon kb/evict-on-demand:origin/kb/evict-on-demand ❯ aws s3 ls --recursive --human-readable --summarize --profile main s3://neon-github-public-dev/build-stats/
2023-02-01 17:26:52  408.2 KiB build-stats/03ef6ec3b798c00a71f79f45c48d1cd388f4b9d5/4065912911/cargo-timing-20230201T152518Z.html
2023-02-01 17:26:52  408.2 KiB build-stats/03ef6ec3b798c00a71f79f45c48d1cd388f4b9d5/4065912911/cargo-timing.html
2023-02-01 17:37:07  403.0 KiB build-stats/9bb4c80aa102b72585bdeb141de41dd613ff088e/4066006107/cargo-timing-20230201T153532Z.html
2023-02-01 17:37:07  403.0 KiB build-stats/9bb4c80aa102b72585bdeb141de41dd613ff088e/4066006107/cargo-timing.html

Total Objects: 4
   Total Size: 1.6 MiB
```